dev/update_mlflow_versions.py

```diff
@@ -0,0 +1,100 @@
+import re
+from pathlib import Path
+from typing import List
+
+import click
+from packaging.version import Version
+
+
+def get_current_version() -> str:
+    text = Path("mlflow", "version.py").read_text()
+    ver = Version(re.search(r'VERSION = "(.+)"', text).group(1))
+    return f"{ver.major}.{ver.minor}.{ver.micro}"
```

Comment:
Could the `.public` property be used here?

```python
return ver.public
```

Reply:
```
>>> Version("1.28.1.dev0").public
'1.28.1.dev0'
```

The `public` property ([implementation](https://github.com/pypa/packaging/blob/8c6a45e20857db912d397afb489832d9528a84c8/packaging/version.py#L354)) doesn't seem to drop the dev suffix.

Reply:
That's right... I forgot that we don't use post markings for dev versions (`+`).